package.json

```diff
@@ -18,6 +19,7 @@
     "i18n-webpack-plugin": "0.2.0",
     "i18next": "^17.0.18",
     "jquery": "^3.3.1",
+    "jspanel4": "^4.9.1",
```

Comment:
Requires `https://babeljs.io/docs/en/babel-plugin-transform-exponentiation-operator` dependency and addition to webpack in order to properly build.